src/Client.php

```diff
@@ -51,16 +52,24 @@ public function request($method, $uri, $data = null, $user = null, $reason = nul
 
     private function _sendRequest($method, $uri, $data = null, $user = null, $reason = null, $comment = null, $additional_headers = null) {
 
+        if (getenv('ENV') === 'local' && isset(self::$mockManager)) {
```

Comment:
Would this impact overall performance, since it would be called in a production environment for each HTTP call?

Also, and I'm unsure if there is a better way to do it, it's a bit unfortunate to have test specific code in the main codepath.